Tests/ValTests/Utils/StaticString+Extensions.swift

```diff
@@ -0,0 +1,8 @@
+extension String {
+
+  /// Creates an instance copying `s`.
```

Comment:
```suggestion
  /// Creates an instance having the value of `s`.
```

Makes it a bit less active; unless the copying is really what you want to emphasize, of course…